tests/nnstreamer_filter_tvm/unittest_filter_tvm.cc

```diff
@@ -14,6 +14,7 @@
 #include <gst/gst.h>
 #include <unittest_util.h>
 
+#include <tensor_common.h>
```

Comment:
IMO this line is unnecessary.

Reply:
The header includes function `gst_tensors_info_free`.

Reply:
proto in nnstreamer_plugin_api.h ?

Reply:
It's test code. Then ok.